src/Ticket.php

```diff
@@ -4831,15 +4831,31 @@ public static function showCentralList($start, $status = "process", bool $showgr
                         break;
 
                     case "process":
-                        $options['criteria'][0]['field']      = 5; // users_id_assign
-                        $options['criteria'][0]['searchtype'] = 'equals';
-                        $options['criteria'][0]['value']      = Session::getLoginUserID();
-                        $options['criteria'][0]['link']       = 'AND';
-
-                        $options['criteria'][1]['field']      = 12; // status
-                        $options['criteria'][1]['searchtype'] = 'equals';
-                        $options['criteria'][1]['value']      = 'process';
-                        $options['criteria'][1]['link']       = 'AND';
+                        $options['criteria'] = [
+                            [
+                                'field'        => 5,
+                                'searchtype'   => 'equals',
+                                'value'        => 'myself',
+                                'link'         => 'OR',
```

Comment:
Shouldn't it be `AND` here ?

```suggestion
                                'link'         => 'AND',
```

Reply:
Yes, thank you :+1:, even if it didn't seem to affect the request.